src/chipeitlib/src/main/java/es/chipeit/lib/extensions/Extensions.kt

```diff
@@ -2,3 +2,10 @@ package es.chipeit.lib.extensions
 
 internal val Byte.Companion.ZERO: Byte
     get() = 0x0
+
+internal fun Int.nonNegativeRem(divisor: Int): Int {
```

Comment:
Remove this ugly ass function as a private one to where it is actually used (GraphicMemory). God help me I hate this so much.